app/components/forms/wizard/basic-details-step.js

```diff
@@ -44,6 +44,10 @@ export default Component.extend(FormMixin, EventWizardMixin, {
     return this.get('settings.isStripeActivated') && find(paymentCurrencies, ['code', this.get('data.event.paymentCurrency')]).stripe;
   }),
 
+  canAcceptOmise: computed('data.event.paymentCurrency', 'settings.isOmiseActivated', function() {
+    return this.get('settings.isOmiseActivated') && find(paymentCurrencies, ['code', this.get('data.event.paymentCurrency')]).omise;
```

Comment:
```js
find(paymentCurrencies, ['code', this.get('data.event.paymentCurrency')]).omise;
```
If you never included omise in currencies, why would this possibly work? Don't make PRs if you haven't tested locally.

Reply:
Sorry , Making changes in dictionary